src/Symfony/Component/Notifier/Message/SmsMessage.php

```diff
@@ -38,7 +39,15 @@ public function __construct(string $phone, string $subject)
 
     public static function fromNotification(Notification $notification, SmsRecipientInterface $recipient): self
     {
-        return new self($recipient->getPhone(), $notification->getSubject());
+        $message = new self($recipient->getPhone(), $notification->getSubject());
+        $message->notification = $notification;
+
+        return $message;
+    }
+
+    public function getNotification(): ?Notification
```

Comment:
As I said in the other PR, I don't like that you can get a Notification from the SMS message, it does not feel right. Instead, if many providers have a "quality" setting, we might add a quality property on the SMS message. Can you see if "big" providers like Nexmo or Twilio also have such a setting?

Reply:
I know, I put thechangehere to open the discussion.  
It appears that nothing equivalent is present in thoses provider.  
They offer an sms pricing without paid options. Or I couldn't find it.

Reply:
That's what I feared unfortunately. Using options on the message is still possible though. So, I think it could be enough for people really wanting to use this feature.

Reply:
ok thanks for yout feedback. 
I might need to clean the mobyt bridge to delete code that won't be used (e.g.: fromNotification in MobytOptions)